src/LocationPickerWidget.js

```diff
@@ -38,12 +117,16 @@ class SelectLocationMap extends Component {
             'icon-allow-overlap': true,
           }}
         >
+          {/*
```

Comment:
we can probably just remove these lines completely instead of having them commented out